.github/ISSUE_TEMPLATE/blog.md

```diff
@@ -0,0 +1,62 @@
+---
+name: Blog Post
+about: Blog website article template
+title: '[BLOG] <blog-title>'
+labels: 'blog'
+assignees: ''
+
+---
+
+<!-- Thank you for contributing to the TAG!
+    Please use this issue template only if you are planning to write a blog post for the TAG Environmental Sustainability.
+    The blog can be found here: https://tag-env-sustainability.cncf.io/blog/
+-->
+
+- **Authors**: *TBD*
+- **TAG ENV Representative**: *TBD*
+- **Reviewers**: *TBD*
+
+<!--
+Authors: anyone can propose to write a blog post which is published to the TAG ENV blog
+TAG ENV Representative: is either a TAG ENV chair or TL
+Reviewers: anyone can be a reviewer of the blog post
+-->
+
+## Outline
+
+<!--
+Please summarize what this blog post is about
+-->
+
+*TBD*
+
+## Scope
+
+<!--
+Please summarize the scope of the blog post
+-->
+
+*TBD*
+
+## Tasks
+
+A more complete summary of tasks are described on XYZ.
+
+- [ ] Outline Blog post
+    - [ ] [**Author**] written a brief outline and the scope of the article so that it can be discussed by the TAG representative and reviewers
+    - [ ] [**TAG ENV Representative**] Blog post outline and theme has been discussed
+- [ ] [**Author**] Blog post written and ready for review
+- [ ] Review
+    - [ ] [**Author**] blog post draft shared blog post via GoogleDocs (public comment)
+    - [ ] [**Author**] notified TAG Community for review via Slack on the `tag-env-sustainability` channel.
+    - [ ] [**Author**] waited for a week after the article has been shared to collect feedback.
```

Comment:
Maybe "Gave no less than a week for feedback.  Note that this could be requested to be extended depending on other current focuses.  As an example, coinciding with KubeCon would require an extension."

Reply:
Thanks, added